src/actions.h

```diff
@@ -28,68 +28,68 @@ using ActionFunction = std::function<bool(Player* player, Item* item, const Posi
 
 class Action : public Event
 {
-	public:
-		explicit Action(LuaScriptInterface* interface);
+public:
+	explicit Action(LuaScriptInterface* interface);
 
-		bool configureEvent(const pugi::xml_node& node) override;
-		bool loadFunction(const pugi::xml_attribute& attr) override;
+	bool configureEvent(const pugi::xml_node& node) override;
+	bool loadFunction(const pugi::xml_attribute& attr) override;
 
-		//scripting
-		virtual bool executeUse(Player* player, Item* item, const Position& fromPosition,
-			Thing* target, const Position& toPosition, bool isHotkey);
+	// scripting
+	virtual bool executeUse(Player* player, Item* item, const Position& fromPosition, Thing* target, const Position& toPosition, bool isHotkey);
 
-		virtual ReturnValue canExecuteAction(const Player* player, const Position& toPos);
-		virtual bool hasOwnErrorHandler() {
-			return false;
-		}
-		virtual Thing* getTarget(Player* player, Creature* targetCreature, const Position& toPosition, uint8_t toStackPos) const;
+	virtual ReturnValue canExecuteAction(const Player* player, const Position& toPos);
+	virtual bool hasOwnErrorHandler()
+	{
+		return false;
+	}
```

Comment:
I'll set it so that inline function definitions will be one liners (if they fit).